src/mlpack/methods/xgboost/loss_functions/sse_loss.hpp

```diff
@@ -29,11 +29,13 @@ namespace ensemble {
 class SSELoss
```

Comment:
I just noticed, we call it `MADGain` and `MSEGain`, but `SSELoss`.  Should we call it `SSEGain` instead?

Reply:
I used "Loss" because everywhere in the boosting literatures (the ones which I have read) they use the term "loss function". Now, calculating gain is one of the things that this loss function do, but it has other things too like residuals, gradients and hessians. So, I thought we can call it `SSELoss`.

Do you think my reasoning for the nomenclature is sound?

Reply:
In general, "gain" is something that is maximized; "loss" is something that is minimized.  Given that we are maximizing, it would make more sense to call it `SSEGain`, in my opinion.  You could add a comment to the class pointing out that we've called it that because our implementation maximizes, instead of minimizes, but honestly it's such a simple mathematical conversion that I doubt anyone would get seriously confused over it.  Some papers talk about gain, some talk about loss, but the two concepts are very related.